AI-and-Analytics/Features-and-Functionality/IntelTensorFlow_ModelZoo_Inference_with_FP32_Int8/README.md

```diff
@@ -24,33 +24,26 @@ The sample intends to help you understand some key concepts:
 |:---             |:---
 | OS              | Ubuntu* 20.04 or higher
 | Hardware        | Intel® Core™ Gen10 Processor <br> Intel® Xeon® Scalable Performance processors
-| Software        | Intel® AI Analytics Toolkit (AI Kit)
+| Software        | Intel® Reference models, Intel Extension for TensorFlow
 
 ### For Local Development Environments
 
-You will need to download and install the following toolkits, tools, and components to use the sample.
+Intel Extension for TensorFlow* is ready for use once you finish the Intel AI Tools installation through AI Tools Selector or Offline Installer.   
+You can refer to the oneAPI AI Tools [product page](https://www.intel.com/content/www/us/en/developer/tools/oneapi/ai-analytics-toolkit.html) for software installation and the *[Get Started with the Intel® AI Tools for Linux*](https://software.intel.com/en-us/get-started-with-intel-oneapi-linux-get-started-with-the-intel-ai-analytics-toolkit)* for post-installation steps and scripts.
 
-- **Intel® AI Analytics Toolkit (AI Kit)**
 
-  You can get the AI Kit from [Intel® oneAPI Toolkits](https://www.intel.com/content/www/us/en/developer/tools/oneapi/toolkits.html#analytics-kit). <br> See [*Get Started with the Intel® AI Analytics Toolkit for Linux**](https://www.intel.com/content/www/us/en/develop/documentation/get-started-with-ai-linux) for AI Kit installation information and post-installation steps and scripts.
-
-  TensorFlow* or Pytorch* are ready for use once you finish installing and configuring the Intel® AI Analytics Toolkit (AI Kit).
-
-### For Intel® DevCloud
-
-The necessary tools and components are already installed in the environment. You do not need to install additional components. See [Intel® DevCloud for oneAPI](https://devcloud.intel.com/oneapi/get_started/) for information.
 
 ## Key Implementation Details
 
-The example uses some pretrained models published as part of the [Model Zoo for Intel® Architecture](https://github.com/IntelAI/models). The example also illustrates how to utilize TensorFlow* and Intel® Math Kernel Library (Intel® MKL) runtime settings to maximize CPU performance on ResNet50 workload.
+The example uses some pretrained models published as part of the [Reference models for Intel® Architecture](https://github.com/IntelAI/models). The example also illustrates how to utilize TensorFlow* runtime settings to maximize CPU performance on ResNet50 workload.
```

Comment:
the new repo name is `Intel® AI Reference models` not `Reference models for Intel® Architecture`, can you please fix it?

Reply:
addressed it accordingly